cogs/Participation.py

```diff
@@ -0,0 +1,91 @@
+from datetime import datetime
+from typing import Optional
+
+import nextcord
+from nextcord import Interaction
+from nextcord import SlashOption
+from nextcord.ext import commands
+from nextcord.utils import get
+
+import apikeys
+import sheets.subsheets as subsheets
+
+
+class Participation(commands.Cog):
+    serverIdList = apikeys.serverIdList()
+    usermap = dict()
+
+    def __init__(self, client):
+        self.client = client
+
+        self.ROLE_FOR_ADMIN_PERMS = "Board"
+        self.NO_PERMS_MSG = "You do not have permission to use this command!"
+
+    @nextcord.slash_command(name="pnew", description="Gives member(s) a participation point. Attaches the event to your user", guild_ids=serverIdList)
+    async def pnew(self, interaction: Interaction, event: str, names: str, date=None, misc=False):
+        if await self.has_permission(interaction):
+            if not date:
+                today = datetime.today()
+                date = str(today.month) + "/" + str(today.day) + "/" + str(today.year)
+            event = event + " " + date
+            await interaction.send("**Processing**")
+            lst = names.split(",")
+            # Remove extraneous spaces.
+            for x in range(len(lst)):
```

Comment:
Use [`str.split()`](https://docs.python.org/3/library/stdtypes.html#str.strip), and possibly a list comprehension:

```py
lst = [name.strip() for name in lst]
```